src/rules/no_var.rs

```diff
@@ -76,37 +75,32 @@ mod tests {
   use crate::test_util::*;
 
   #[test]
-  fn no_var_invalid() {
-    assert_lint_err::<NoVar>(
-      r#"var someVar = "someString"; const c = "c"; let a = "a";"#,
-      0,
-    );
+  fn no_var_valid() {
+    assert_lint_ok::<NoVar>(r#"let foo = 0; const bar = "bar""#);
+  }
```

Comment:
Just one more request - could you also replace this `assert_lint_ok` function with `assert_lint_ok!` macro? like https://github.com/denoland/deno_lint/blob/master/src/rules/no_namespace.rs#L62-L68

Reply:
Done